tests/integration/baseImage_test.go

```diff
@@ -0,0 +1,97 @@
+package integration
+
+import (
+	"fmt"
+	"github.com/lf-edge/eden/pkg/controller/einfo"
+	"github.com/lf-edge/eve/api/go/config"
+	"testing"
+	"time"
+)
+
+//TestBaseImage test base image loading into eve
+func TestBaseImage(t *testing.T) {
+	ctx, err := controllerPrepare()
+	if err != nil {
+		t.Fatal("Fail in controller prepare: ", err)
+	}
+	var baseImageTests = []struct {
```

Comment:
Are we going to add more test cases here such as a downgrade or subsequent update?
If so a map from testname string to the struct means you can log the testname in the loop below.

Reply:
Good question @eriknordmark -- I hope we will. At this point I'm just trying to get the infrastructure rock-solid and the tests themselves are there mostly to exercise the infrastructure. They are not meant to be "definitive".

I guess what I'm trying to say is this: the part of the test that we all should pay attention to is the flow of how it calls the underlying Eden library to drive the controller, etc. The rest we can fix/evolve later.

Agreed?